packages/components/src/Chip/components/ChipDismissible/ChipDismissible.test.tsx

```diff
@@ -21,14 +21,14 @@ describe("Chip Dismissible", () => {
     expect(container.querySelector("svg")).toBeInTheDocument();
   });
 
-  it("calls onClick when clicked", () => {
+  it("does not call onClick when clicked because suffix is clickable", () => {
```

Comment:
Should we also test the opposite? `calls onClick when the suffix is clicked`

Reply:
Lol yup. Thanks for keeping me honest. The functionality changed (comment from Chris) and I was too lazy to go add a testId to the Suffix so I could grab it in the test. I've done that now, and the test is passing. Great piece of feedback Taylor! Good eye.

Reply:
love it! thank you!